awx/ui_next/src/components/JobList/useThrottle.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { mount } from 'enzyme';
+import useThrottle from './useThrottle';
+
+function TestInner() {
+  return <div />;
+}
+function Test({ value }) {
+  const throttled = useThrottle(value, 500);
+  return <TestInner throttled={throttled} />;
+}
+
+jest.useFakeTimers();
+
+describe('useThrottle', () => {
+  test.skip('should throttle value', async () => {
```

Comment:
did you intend to skip this test for some reason?

Reply:
I wasn't able to get this test working, because it required stubbing out the Date object. I may have to delete it :/

Reply:
Yeah, I'd rather have a deleted test, than a skipped test IMO